### ISSUE TITLE
Improve on WCAG "Operable" category - a11y audit

### ISSUE THREAD

**Issue by xmedr** [Metro-Records/la-metro-councilmatic#1022]
Go through the following criteria for [WCAG's Operable section](https://www.w3.org/TR/WCAG21/#operable) and improve their issues to meet WCAG requirements:

### Level A
- 2.1.1 Keyboard:
  - [ ] Mostly fine except for the maps, which currently aren't all the way keyboard interactive. Will have to reevaluate if the fact that there is an alternative for interacting with the map (the members listing table) is enough to bypass this

- 2.5.3 Label in Name:
  - [ ] The board report search button has no text or name
  - [ ] The agenda pdf form's submit button should have an `aria-label` with the same text that it displays
  
### Level AA
- 2.4.5 Multiple Ways:
  - [ ] Mostly okay, with the caveat that I haven't seen UI that takes you to the login page or the archive search page within the app (not the external one that's in the navbar). If this is by design, then it might be okay.
  
- 2.4.6 Headings and Labels:
  - [ ] We definitely skip heading levels throughout the app for aesthetic purposes. We should change them to be in a specific order, and then use css classes to style them how we want.
  - [ ] [Consider adding <caption> to appropriate tables](https://www.w3.org/WAI/tutorials/tables/caption-summary/). It's not a requirement but could be nice to do where we can.

- 2.4.7 Focus Visible:
  - [ ] The maps on the board members listing and detail pages lose visible focus in a few places. This may interfere with 2.4.3 Focus Order, if those elements that are being focused, shouldn't be focusable in the first place.
  - [ ] On the board report search component, navigating over the radio buttons loses visible focus
  - [ ] Links should have some kind of indication they're being focused on.
  
- Extra:
  - [ ] Need to identify which links open in a new tab/window. Some links like those on a [legislation detail page](https://lametro.datamade.us/board-report/2023-0426/) have an icon for this. Maybe we can add context to that icon and then add that to every `target=”_blank”` link that doesn’t already have one

Connects #1012

**Comment by xmedr** [comment]
work on this is done and merged into the overall style/a11y branch featured in pr #1009. will ask for a metro review once the first round of a11y changes are approved

**Comment by xmedr** [comment]
Closed by #1051
